apps/spruce/src/utils/string/index.ts

```diff
@@ -1,9 +1,16 @@
 import { format, toZonedTime } from "date-fns-tz";
 import get from "lodash/get";
+import {
+  shortenGithash,
+  trimStringFromMiddle,
+  copyToClipboard,
+} from "@evg-ui/lib/utils/string";
```

Comment:
We should not be importing these functions here and we should instead just import the functions when needed